docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -14,19 +21,29 @@ All the steps in the graphics pipeline go from top to bottom and are shown below
 As you can see, each stage in the pipeline takes in the previous stage's output as the input, the rectangle blocks are pipeline stages that are not programmable but are configurable, while the rounded rectangle blocks are stages that are fully programmable.
 To draw most of the things throughout this series we'll mostly need these stages: Input Assembler, Vertex Shader and the Pixel Shader, Output Merger.
 
-The Vertex and Pixel shaders are fully programmable, and we'll write a very basic program for them.
+The Vertex and Pixel shaders are fully programmable and we'll write a very basic program for them.
 
-The other two stages are not programmable, but they are fairly easy to understand and configure, the **Input Assembler** is responsible for processing the vertices in an eventual vertex buffer into the primitive of our choice, which is of course, triangles in our case, and sending this processed output to the Vertex Shader. The **Output Merger** instead is responsible for combining the values written by the pixel shader, may that be depth, color or other things, into the one or more render targets that we provide to the OM, we only have one render target for now.
+The other two stages are not programmable but they are fairly easy to understand and configure:
 
-### Vertex Shader
+- the **Input Assembler** is responsible for processing the vertices in an eventual vertex buffer into the primitive topology of our choice, which in our case is a form of triangles, and sending this processed output to the Vertex Shader to process. 
```

Comment:
maybe simpler is a more accurate (since we will get into specifics later): `processed output to be processed again - but this time by our Vertex Shader.

this also highlights how we input to prepare for our own logic to do as we please with data